lib/utils.bash

```diff
@@ -156,10 +156,12 @@ get_version_in_dir() {
   local legacy_filenames=$3
 
   local asdf_version
-  asdf_version=$(parse_asdf_version_file "$search_path/.tool-versions" "$plugin_name")
+
+  file_name=$(version_file_name)
+  asdf_version=$(parse_asdf_version_file "$search_path/$file_name" "$plugin_name")
 
   if [ -n "$asdf_version" ]; then
-    printf "%s\\n" "$asdf_version|$search_path/.tool-versions"
+    printf "%s\\n" "$asdf_version|$search_path/$file_name"
```

Comment:
This `get_version_in_dir` function should also read `ASDF_DEFAULT_TOOL_VERSIONS_FILENAME` if it is set.